src/main/resources/assets/crimson/lang/en_us.json

```diff
@@ -21,10 +21,13 @@
   "block.crimson.realgar_ore": "Realgar Ore",
   "block.crimson.vanadinite_ore": "Vanadinite Ore",
   "block.crimson.crimson_thorn": "Crimson Thorn",
+	"block.crimson.potted_crimson_thorn": "Potted Crimson Thorn",
   "block.crimson.crimson_log": "Crimson Log",
   "block.crimson.crimson_leaves": "Crimson Leaves",
-  "block.crimson.crimson_sapling": "Crimson Sapling",
-  "block.crimson.crimson_tendrils": "Crimson Tendrils",
+	"block.crimson.crimson_sapling": "Crimson Sapling",
```

Comment:
same as above

Reply:
Fixed in 9f64c8574f19b61a662f9ecfd49a927533642067.